src/internal/connector/discovery/api/users.go

```diff
@@ -220,3 +326,31 @@ func validateUser(item any) (models.Userable, error) {
 
 	return m, nil
 }
+
+func convertInterfaceToString(ctx context.Context, data interface{}, userInfo *UserInfo) string {
```

Comment:
nit on func name.  also, prefer `any` over `interface{}`.
```suggestion
func toString(ctx context.Context, data any, userInfo *UserInfo) string {
```